build-system/tasks/performance/helpers.js

```diff
@@ -172,7 +172,7 @@ function getLocalVendorConfig(vendor) {
  * @return {!Promise<string>} Resolves with relative path to file
  */
 async function getFileFromAbsolutePath(filePath) {
-  return fs.readFileSync(filePath);
+  return fs.readFileSync(filePath).toString();
```

Comment:
Docs say `If no encoding is specified (using options.encoding), the data is returned as a <Buffer> object. Otherwise, the data will be a string.` - since you need to make a change here, I think it's better to set an encoding

Reply:
Also in build-system/tasks/performance/print-report.js

Reply:
Yep, best to just pass in an encoding. Also, agree that we should use the async version. 

```suggestion
  return await fs.readFile(filePath, `utf-8`);
```